ctapipe/tools/display_integrator.py

```diff
@@ -226,24 +226,19 @@ class DisplayIntegrator(Tool):
     ).tag(config=True)
     channel = Enum([0, 1], 0, help="Channel to view").tag(config=True)
 
-    aliases = Dict(
-        dict(
-            f="EventSource.input_url",
-            max_events="EventSource.max_events",
-            E="DisplayIntegrator.event_index",
-            T="DisplayIntegrator.telescope",
-            C="DisplayIntegrator.channel",
+    aliases = {
+        ("i", "input"): "EventSource.input_url",
+        ("m", "max-events"): "EventSource.max_events",
+        "E": "DisplayIntegrator.event_index",
+        "T": "DisplayIntegrator.telescope",
+        "C": "DisplayIntegrator.channel",
+    }
+    flags = {
+        "id": (
+            {"DisplayDL1Calib": {"use_event_index": True}},
```

Comment:
This should also be a flag, right?

Reply:
Was not sure about that one, but yes, why not